### DISCUSSION TITLE
第三者レビュー ログ-方式

### DISCUSSION THREAD

**Discussion by creature-water-valley** [ws-4020/mobile-app-crib-notes#569] [General]
#### アプリのログ出力

- アプリのログ管理の仕組み
  - 最後のほうで「モバイルアプリ」という言葉を使っていましたが、「アプリ」との使い分けが気になりました。使い分けがないなら資料中は統一した方がよさそうに思いました。

**Comment by creature-water-valley** [comment]
### 対応内容

- アプリで統一する

**Comment by kozake** [comment]
### 対応PR

https://github.com/ws-4020/mobile-app-crib-notes/pull/564

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
